pupil_src/shared_modules/gaze_producer/gaze_from_offline_calibration.py

```diff
@@ -0,0 +1,92 @@
+"""
+(*)~---------------------------------------------------------------------------
+Pupil - eye tracking platform
+Copyright (C) 2012-2018 Pupil Labs
+
+Distributed under the terms of the GNU
+Lesser General Public License (LGPL v3.0).
+See COPYING and COPYING.LESSER for license details.
+---------------------------------------------------------------------------~(*)
+"""
+from gaze_producer.controller.reference_location_controllers import (
+    ReferenceDetectionController,
+    ReferenceEditController,
+)
+from gaze_producer.model.reference_location_storage import ReferenceLocationStorage
+from gaze_producer.ui.reference_location_renderer import ReferenceLocationRenderer
```

Comment:
Personally I would prefer it if the ui-namespace survived the import command to add more context. 
I.e. `from gaze_producer import ui`
and then later instantiate `ui.ReferenceLocationRenderer`